src/main/webapp/app/modules/canvas/visualization/canvas.tsx

```diff
@@ -247,7 +247,7 @@ const Canvas = (props: VisualizationProp) => {
 
   return (
     <>
-      {props.isSocketConnected && <FilterPanel />}
+      {props.isSocketConnected && <FilterPanel {...props} />}
```

Comment:
whole props is passed...pass selected data

Reply:
done